locust/runners.py

```diff
@@ -760,7 +778,7 @@ def _wait_for_workers_report_after_ramp_up(self) -> float:
 
         assert False, "not supposed to reach that"
 
-    def stop(self):
```

Comment:
"not supposed to reach that"-assert on 779 is unnecessary, change  the if statements to if-else and remove this.

Reply:
Done in the other PR